client/client-multi/client-c/Modules/doxygen/CMakeLists.txt

```diff
@@ -0,0 +1,44 @@
+#
+# Copyright 2014-2016 CyberVision, Inc.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+
+# This cmake list is resposible for building doxygen documentation.
+
+include (FindDoxygen)
+
+if(DOXYGEN_FOUND)
+   # Build documentation for each directory
+   set(INPUT_DIRECTORY ${KAA_SDK_DIR}/src)
+   set(OUTPUT_DIRECTORY ${KAA_SDK_DIR}/target/apidocs/doxygen)
+   set(HTML_EXTRA_FILES ${KAA_SDK_DIR}/README)
```

Comment:
You shouldn't set HTML_EXTRA_FILES from there. CMake should only set source directory (`@KAA_SDK_DIR@`) and output directory (`@OUTPUT_DIRECTORY@`)